src/utils/lookup-table-utils.ts

```diff
@@ -1,36 +1,49 @@
 import { DataTable } from '../entities/dataset/data-table';
 import { LookupTable } from '../entities/dataset/lookup-table';
 import { DataTableDescription } from '../entities/dataset/data-table-description';
-import { SupportedLanguagues } from '../enums/locale';
+import { Locale } from '../enums/locale';
 import { MeasureLookupPatchDTO } from '../dtos/measure-lookup-patch-dto';
 import { LookupTablePatchDTO } from '../dtos/lookup-patch-dto';
 
 import { logger } from './logger';
+import { Database } from 'duckdb-async';
+import { Dataset } from '../entities/dataset/dataset';
+import { ViewErrDTO } from '../dtos/view-dto';
+import { viewErrorGenerators } from './view-error-generators';
+import { SUPPORTED_LOCALES } from '../middleware/translation';
+import { MeasureLookupTableExtractor } from '../extractors/measure-lookup-extractor';
+import { DataValueFormat } from '../enums/data-value-format';
+import { t } from 'i18next';
 
-export function convertFactTableToLookupTable(factTable: DataTable) {
+export function convertDataTableToLookupTable(dataTable: DataTable) {
   const lookupTable = new LookupTable();
-  lookupTable.id = factTable.id;
-  lookupTable.fileType = factTable.fileType;
-  lookupTable.filename = factTable.filename;
-  lookupTable.mimeType = factTable.mimeType;
-  lookupTable.hash = factTable.hash;
+  lookupTable.id = dataTable.id;
+  lookupTable.fileType = dataTable.fileType;
+  lookupTable.filename = dataTable.filename;
+  lookupTable.mimeType = dataTable.mimeType;
+  lookupTable.hash = dataTable.hash;
   return lookupTable;
 }
 
 export function columnIdentification(info: DataTableDescription) {
-  let lang = 'zz';
-  for (const supLang of Object.values(SupportedLanguagues)) {
-    if (info.columnName.toLowerCase().endsWith(supLang.code.toLowerCase())) {
-      lang = supLang.code;
-      break;
-    } else if (info.columnName.toLowerCase().endsWith(supLang.name.toLowerCase())) {
-      lang = supLang.code;
+  let columnLang = 'zz';
```

Comment:
zz?

Reply:
`zz` Is deliberately not a language code and is reserved against the standard.  If you end up with `zz` mean we weren't actually able to match the language used.  So later on in the process we just end up ignoring this columns.